lib/index.js

```diff
@@ -51,6 +51,9 @@ function loadString(db, data, opts, callback) {
 
     return db.info();
```

Comment:
another fix is to just join the next `then()` to the previous `db.info()` which I think is cleaner